distribution/lib/Standard/Test/0.0.0-dev/src/Bench.enso

```diff
@@ -20,21 +20,22 @@ type Bench
              example_measure =
                  Bench.measure Examples.get_boolean "foo" iter_size=2 num_iters=1
     measure : Any -> Text -> Integer -> Integer -> Nothing
-    measure = ~act -> label -> iter_size -> num_iters ->
+    measure = ~act -> label -> iter_size -> num_iters  ->
+        dry_run = Environment.get_or_else "ENSO_BENCHMARK_TEST_DRY_RUN" "False" == "True"
         result = Ref.new 0.0
         single_call = _ ->
             x1 = System.nano_time
             Runtime.no_inline act
             x2 = System.nano_time
             x2 - x1
-        iteration = it_num ->
+        iteration = it_size -> it_num ->
             act_it_num = num_iters - it_num
-            res = times iter_size single_call
+            res = times it_size single_call
             avg = avg_list res
             fmt = (avg / 1000000).format "%.2f"
             result.put (result.get + avg)
             IO.println (label + "/iteration:" + act_it_num.to_text + ": " + fmt + "ms")
```

Comment:
If I understand correctly, the way this is currently done - in dry-run mode it will print `lavel/iteration:999: 123ms` - i.e. instead of iteration `0` it will print some high number of iteration becaue `it_num` will be equal to `1` but `num_iters` will still be a non-dry-run number.

However, instead of 'fixing' this, I'd suggest to completely change the output here to indicate explicitly that we are running a dry-run - because currently there is no indication as to that.

Reply:
True, only noticed that now